Integrations/Gmail/Gmail.py

```diff
@@ -289,46 +375,132 @@ def mail_to_incident(msg, service, user_key):
             'path': file_result['FileID'],
             'name': attachment['Name'],
         })
+    # date in the incident itself is set to GMT time, the correction to local time is done in Demisto
+    gmt_time = move_to_gmt(parsed_msg['Date'])
 
-    return {
+    incident = {
         'type': 'Gmail',
         'name': parsed_msg['Subject'],
         'details': parsed_msg['Body'],
         'labels': create_incident_labels(parsed_msg, headers),
-        'occurred': parse_time(parsed_msg['Date']),
+        'occurred': gmt_time,
         'attachment': file_names,
         'rawJSON': json.dumps(parsed_msg),
     }
+    return incident
+
+
+def organization_format(org_list):
+    if org_list is None or len(org_list) == 0:
+        return None
+    org_str = ''
+    for org in org_list:
+        if org.get('name') is None:
+            continue
+
+        else:
+            org_str = org_list + str(org.get('name')) + ','
+
+    return org_str.rstrip(',')
 
 
 def users_to_entry(title, response):
     context = []
+
     for user_data in response:
+        username = user_data.get('name').get('givenName') if user_data.get('name') \
+            and 'givenName' in user_data.get('name') else None
+
+        display = user_data.get('name').get('fullName') if user_data.get('name') \
+            and 'fullName' in user_data.get('name') else None
+
         context.append({
             'Type': 'Google',
             'ID': user_data.get('id'),
-            'UserName': (user_data.get('name').get('givenName')
-                         if user_data.get('name') and 'givenName' in user_data.get('name') else None),
-            'DisplayName': (user_data.get('name').get('fullName')
-                            if user_data.get('name') and 'fullName' in user_data.get('name') else None),
+            'UserName': username,
+            'DisplayName': display,
             'Email': {'Address': user_data.get('primaryEmail')},
             'Gmail': {'Address': user_data.get('primaryEmail')},
             'Group': user_data.get('kind'),
             'CustomerId': user_data.get('customerId'),
+            'Groups': user_data.get('kind'),
+            'Domain': user_data.get('primaryEmail').split('@')[1],
+            'Username': username,
```

Comment:
move this next to original and add a comment specifying this is new standard context